_drafts/circle-two-point-oh.md

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
This file needs to be moved into the `_posts` directory and renamed to match the convention there (for Jekyll).